.github/workflows/python.yml

```diff
@@ -104,10 +109,9 @@ jobs:
           # Below will exit with non-zero status if model changes are missing migrations
           sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
 
-  test-py3:
-    name: 'py3 [ignore fails]'
+  test-py3-simple:
+    name: 'python3.6 backend (simple)'
```

Comment:
What's simple? I think we should just call this the same as what we call it in travis. So `Backend` or `Backend with migrations`

Reply:
👍 then I'll figure out BYK's change and then merge